### ISSUE TITLE
Challengers and ITF tournaments results

### ISSUE THREAD

**Issue by rjcpinto** [mcekovic/tennis-crystal-ball#328]
First of all, awsome work you have been doing with Ultimate Tennis Statistics!

However, I would like to know if you are considering adding ATP Challenger and ITF tournaments results? I think it would be a great addition to the database.

Thanks!

**Comment by mcekovic** [comment]
Yes, this is in the plan, but it is a lot of work, not only to include historical results and organize data updates, but also to update the web site so that stats can be seen with or w/o them (pure ATP main draw stats, like shown now).
So this is on hold for now.

**Comment by rjcpinto** [comment]
Good to year it! Despite understanding it takes some work, I think the addition of this feature will take the site to a higher level and make it the reference in relation to tennis statistics. 
I'm looking forward to it! Thank you.
